### PR TITLE
upgrade_tensorflow_to_latest_package

### PR DESCRIPTION
**Pull request by zhengkai6**
1. upgrade package to 1.3.0rc2 the latest package
2. imporve ipykernel install steps under python 2.x

### CODE REVIEW

**Comment by xixikaikai** [comment]
RE: I signed it!

**Comment by xixikaikai** [comment]
RE: Updated

File: tensorflow/tools/docker/Dockerfile [hunk 1]
```diff
@@ -44,7 +45,7 @@ RUN pip --no-cache-dir install \
 
 # Install TensorFlow CPU version from central repo
 RUN pip --no-cache-dir install \
-    http://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.0.0-cp27-none-linux_x86_64.whl
+    http://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-1.3.0rc2-cp27-none-linux_x86_64.whl
```
Comment (caisq) [inline tensorflow/tools/docker/Dockerfile]: These lines are automatically edited during the build processed described at:
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/docker#rebuilding-the-containers

File: tensorflow/tools/docker/Dockerfile [hunk 2]
```diff
@@ -23,10 +23,11 @@ RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
     python get-pip.py && \
     rm get-pip.py
 
+RUN python2 -m pip install ipykernel
```
Comment (caisq) [inline tensorflow/tools/docker/Dockerfile]: Can you help me understand why these changes related to ipykernel install is needed?

Comment (xixikaikai) [reply tensorflow/tools/docker/Dockerfile]: I turned to this page http://ipython.readthedocs.io/en/stable/install/kernel_install.html, and when I build using the org. one, the errors appeared, and the image can not be built correctly.

File: tensorflow/tools/docker/Dockerfile (hunk 1 above)
Comment (xixikaikai) [reply tensorflow/tools/docker/Dockerfile]: OK, you mean that 0.0.0 auto pointed to the latest version of cp2?

**Comment by xixikaikai** [comment]
I signed it!

**Comment by drpngx** [comment]
I can't find your github username in our CLA database. Could you double check?

File: tensorflow/tools/docker/Dockerfile (hunk 1 above)
Comment (caisq) [reply tensorflow/tools/docker/Dockerfile]: No. I meant please do not change this line :)

File: tensorflow/tools/docker/Dockerfile (hunk 2 above)
Comment (caisq) [reply tensorflow/tools/docker/Dockerfile]: We have nightly builds on TensorFlow Jenkins that are based on this file. They have been consistently passing in recent days. Please see the log at:
http://ci.tensorflow.org/view/Nightly/job/nightly-docker-cpu/

The console logs contain the commands used in the builds.
